scripts/run-sphinx.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# generate code documentation via sphinx and upload to rtd
+
+SCRIPT_PATH=$(dirname $BASH_SOURCE)
+AMICI_PATH=$(cd $SCRIPT_PATH/.. && pwd)
+
+cd ${AMICI_PATH}/documentation
+source ../build/venv/bin/activate
+pip3 install sphinx nbsphinx recommonmark sphinx_rtd_theme petab sphinx-autodoc-typehints
+
+# add linebreaks that swig is missing, we can repeat this as many times as
+# we want
+AMICI_FILE=${AMICI_PATH}/python/sdist/amici/amici.py
+sed -i -e -E $'s/([ ]+):(rtype|type|param|return)/\\\n\\1:\\2/g' $AMICI_FILE
```

Comment:
What went wrong there? :)

Reply:
swig conversion of docstrings is not as good as you want and seemingly arbitrarily misses linebreaks from time to time 🤷‍♂